BlockFrenzy/GameScene.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  GameScene.swift
+//  BlockFrenzy
+//
+//  Created by McLoughlin David J. on 3/8/18.
+//  Copyright © 2018 McLoughlin David J. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+import AVFoundation
+
+class GameScene: SKScene {
+    
+    let BallCatName = "ball"
```

Comment:
Names of variables or constants should be lowercase. Only class names are uppercase.